web/src/components/tasks/business-formation/contacts/ContactsStep.test.tsx

```diff
@@ -369,6 +375,51 @@ describe("Formation - ContactsStep", () => {
       expect(page.getInputElementByLabel("Signer 1").value).toBe("signer 2");
       expect(page.getInputElementByLabel("Signer 2").value).toBe("signer 3");
     });
+
+    describe("signers", () => {
+      it("displays error to both enter name and sign if both are missing", async () => {
```

Comment:
Can we update description to  `"displays error to both enter name and signature if both are missing"`?